examples/new_algorithms/reinforce/reinforce.sh

```diff
@@ -33,6 +33,7 @@ TRIAL_NAME=$MODEL_FAMILY-$MODE-manual
 
 # The following command shows an example of manual allocation on two nodes,
 # but it can be modified according to the specific model and the available GPUs.
+unset CLUSTER_SPEC_PATH
```

Comment:
Is this necessary? CLUSTER_SPEC_PATH is still required to change fileroot in local mode

Reply:
Yep, because this script is in the local setting and CLUSTER_SPEC_PATH will change the node prefix.